content/img/illustrations/illus--translation-connection.svg

```diff
@@ -0,0 +1,75 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
You'll need to use the same baseline for your image to be aligned with the images of other connections. Look at the guide index for example. Image should be vertically aligned, on the same baseline.

Reply:
Yes, I'll try to change this image...